### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #810

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -515,16 +515,22 @@ patch updates are routinely and regularly released.
...
-3) Fetch the tag from the upstream repo, and checkout this tag.
-   Create the distribution tarball using ``python setup.py sdist``,
-   and then upload the resulting tarball to PyPI.
+3) Fetch the tag from the upstream repo, and checkout this tag.  Create the
+   distribution tarball using ``python setup.py sdist``, and then upload the
+   resulting tarball to PyPI using ``twine upload
+   dist/ga4gh-MAJOR.MINOR.PATCH.tar.gz`` (of course, using the correct file
+   name)
 4) Verify that the documentation at
-   http://ga4gh-reference-implementation.readthedocs.org/en/stable/
```
Comment (dcolligan) [inline docs/development.rst]: @jeromekelleher `stable` has still not updated... is this a problem?

Comment (jeromekelleher) [inline docs/development.rst]: Odd, I had to trigger the build for stable manually. It seems to be fine now. Have you got an account for RTD? I'll add you as a maintainer too so we don't have a single point of failure.

Anyway, this should still point to ``stable`` not latest. In RTD, latest==master, and stable==last tagged version

**Comment by dcolligan** [comment]
@jeromekelleher updated

my RTD name is `dannycolligan`

**Comment by jeromekelleher** [comment]
Thanks, +1.

You've been added as a maintainer on RTD. @kozbo, you should probably be added there too; do you have a readthedocs ID?

**Pull request merged**